nixos/doc/manual/configuration/linux-kernel.xml

```diff
@@ -27,14 +29,13 @@ zcat /proc/config.gz
 linkend="sec-customising-packages" />). For instance, to enable support
   for the kernel debugger KGDB:
 <programlisting>
-nixpkgs.config.packageOverrides = pkgs:
-  { linux_3_4 = pkgs.linux_3_4.override {
-      extraConfig =
-        ''
-          KGDB y
-        '';
-    };
+nixpkgs.config.packageOverrides = pkgs: pkgs.lib.recursiveUpdate pkgs {
+  linuxKernel.kernels.linux_3_4 = pkgs.linuxKernel.kernels.linux_3_4.override {
```

Comment:
should be fine to bump 3_9 with a kernel actually available.

Reply:
bumped to linux_5_10